tests/conftest.py

```diff
@@ -14,15 +14,60 @@ def montage():
 
 @pytest.fixture(scope="session")
 def raw():
-    """Fixture for physionet EEG subject 4, dataset 1."""
+    """Return an `mne.io.Raw` object for use with unit tests.
+
+    This fixture downloads and reads in subject 4, run 1 from the Physionet
+    BCI2000 (eegbci) open dataset. This recording is quite noisy and is thus a
+    good candidate for testing the PREP pipeline.
+
+    File attributes:
+    - Channels: 64 EEG
+    - Sample rate: 160 Hz
+    - Duration: 61 seconds
+
+    This is only run once per session to save time downloading.
+
+    """
     mne.set_log_level("WARNING")
-    # load in subject 1, run 1 dataset
+
+    # Download and read S004R01.edf from the BCI2000 dataset
     edf_fpath = eegbci.load_data(4, 1, update_path=True)[0]
+    raw = mne.io.read_raw_edf(edf_fpath, preload=True)
+    eegbci.standardize(raw)  # Fix non-standard channel names
+
+    return raw
+
 
-    # using sample EEG data (https://physionet.org/content/eegmmidb/1.0.0/)
+@pytest.fixture(scope="session")
+def raw_clean(montage):
+    """Return an `mne.io.Raw` object with no bad channels for use with tests.
+
+    This fixture downloads and reads in subject 1, run 1 from the Physionet
+    BCI2000 (eegbci) open dataset, interpolates its two bad channels (T10 & F8),
```

Comment:
is there perhaps a file from that dataset that is clean to begin with? ... If you don't wanna test all of them, that's fine though. This is just a side note.

Reply:
Out of over 200 recordings, the script found 6 that were totally clean:

```
Clean recordings:
 - Subject 24 Run 2
 - Subject 30 Run 2
 - Subject 38 Run 1
 - Subject 58 Run 2
 - Subject 74 Run 2
 - Subject 91 Run 2
```

Glad I didn't try to do this manually!

Reply:
Fixed in the latest commit! For future reference, S024R02 had some sub-threshold high noise channels that broke the HF noise detection test. The median channel noisiness for S030R02 was like 30 times lower, so I used that instead.